conjur/input_conjurrc.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+"""
+InputConjurrc module
+
+This module represents an object that holds conjurrc data
+"""
+# pylint: disable=line-too-long, too-few-public-methods
+class InputConjurrc:
+    """
+    Used for setting user input data
+    """
+    def __init__(self, url = None, account = None, cert = None):
+        self.appliance_url = url
+        self.account = account
+        self.cert_file = cert
+        self.plugins = []
+
+    def __repr__(self):
+        return "{'appliance_url': %r, 'account': %r, 'cert_file': %r, 'plugins': %r}" % (self.appliance_url,
```

Comment:
convert to fstring

Reply:
I need those `{` `}` to represent an object. Not sure I can with fstring